client/components/App.js

```diff
@@ -42,14 +42,25 @@ const App = () => {
     setEndTime(string);
   };
 
+ // useLocation hook to get the current pathname
+ const location = useLocation();
+ const isAdminPage = location.pathname.startsWith('/admin');
+
   return (
-      <div className='body'>
-        <div>
-          <Link to="admin/login">Admin Log in</Link>
-          <h1>TimeCroc</h1>
-          <Clock />
-        </div>
-        
+    <div className='body'>
+      <div>
+      {/* Render login button only if not on an admin page */}
+        {!isAdminPage && (
+         <Link to="admin/login"><button className='login-btn'>Admin Log in</button></Link>
+        )}
+      {/* Render sign out button only if admin is logged in */}
+        {isAdminLoggedIn ? (
+          <button className='sign-out-btn' onClick={() => setIsAdminLoggedIn(false)}>Sign Out</button>
```

Comment:
When I click this button, the route stays at /admin and I'm stuck looking at a blank page with no functionality.
<img width="813" alt="Screen Shot 2023-04-06 at 3 38 29 PM" src="https://user-images.githubusercontent.com/71556587/230490139-1baf476b-35d3-462a-9a9b-d044e96d8b8d.png">
Can we add something to this onClick to route back to the root? 
When an Admin user logs out, I'd like the user to be routed back to the main page of the application.

Reply:
I made a new ticket in Jira (TC-14) to address this issue. We're going to create a LogOutButton component that makes a call to the backend to properly log the admin out and then redirect them back to the home page.